Source/Core/Core/NetPlayProto.h

```diff
@@ -9,6 +9,9 @@
 #include "Common/CommonTypes.h"
 #include "Core/HW/EXI/EXI_Device.h"
 
+#define LZO_IN_LEN 1024 * 64
```

Comment:
These should be concrete constants, not defines. It's likely preferable to prefix these with `NETPLAY_` as well.